ci/docker/kpsrtutorial-1.0.0.dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ghcr.io/klepsydra-technologies/kpsrbase:1.0.0
+
+# Install kpsr-core
+ARG KPSRCOREVERSION="v8.0.0"
+RUN git clone https://github.com/klepsydra-technologies/kpsr-core.git && \
+    cd kpsr-core && \
+    git checkout ${KPSRCOREVERSION} && \
+    git submodule update --init --recursive && \
+    mkdir build && cd build && \
+    cmake -DCMAKE_BUILD_TYPE=Release -DKPSR_WITH_DOXYGEN=true -DKPSR_WITH_DDS=false -DKPSR_WITH_ZMQ=false -DKPSR_TEST_PERFORMANCE=true -DKPSR_WITH_SOCKET=true -DKPSR_WITH_YAML=true -DKPSR_WITH_CODE_METRICS=true ../ && \
+    make -j$(nproc) && \
+    sudo make install && \
+    cd ../../ && \
+    rm -rf kpsr-core
+
+# Install Python package build needed by kpsr-codegen
+RUN pip3 install build
+# Install kpsr-codegen
+ARG KPSRCODEGENVERSION="main"
+RUN git clone https://github.com/klepsydra-technologies/kpsr-codegen.git && \
+    cd kpsr-codegen && \
+    git checkout ${KPSRCODEGENVERSION} && \
+    sed -i 's/git@github.com:/https:\/\/github.com\//g' .gitmodules && \
+    git submodule update --init --recursive && \
+    python3 -m build --outdir dist && \
+    pip3 install dist/kpsr_codegen-1.0-py3-none-any.whl && \
+    cd ../../ && \
+    rm -rf kpsr-codegen
+
+# Install kpsr-tutorial
+ARG KPSRTUTORIALVERSION="kce-74"
```

Comment:
Because of the build problems found in https://github.com/klepsydra-technologies/kpsr-tutorial/pull/38 I had to for now use this branch to not have build problems for kpsr-tutorial.

In case the PR is merged before this I could change the branch back to `main`.